Windows build fails

G:\brewtarget\src\brewtarget.cpp: In static member function 'static const QDir Brewtarget::getConfig
Dir()':
G:\brewtarget\src\brewtarget.cpp:358:8: error: 'success' was not declared in this scope
    if( success != 0 )
        ^

Looks like one of our refactors didn't clean up the windows branches.

--- a/src/brewtarget.cpp
+++ b/src/brewtarget.cpp
@@ -355,8 +355,6 @@ const QDir Brewtarget::getConfigDir()
    // Now we should be in the base directory (i.e. Brewtarget-2.0.0/)
 
    dir.cd("data");
-   if( success != 0 )
-      *success = true;
    return dir.absolutePath() + "/";
 
 #else

